### ISSUE TITLE
[A11Y] aria-current on active menu item

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#6234]
## Description of the bug

An accessibility review on a site in development called out the following:

> No aria dictating the "active" element in the navigation for a non-visual user using a screen reader

The solution?

> `aria-current="page"` on the active `<a>`

 - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current

**Comment by indigoxela** [comment]
That totally makes sense to me, many thanks for opening the issue and the PR. :+1: 

However, functional tests are failing (unexpected markup, I guess). And another thing: "Fail: Image linked to content formatter displaying correctly on full node view."

Could it be, that the current solution is a bit too greedy, possibly misleading? An image field linked to content shouldn't get the aria-current attribute, I guess. Or should it? But I didn't debug, so the test just might fail because of unrelated stuff.

**Comment by indigoxela** [comment]
Played with the sandbox, and I think, this works as expected.

The menu items, the primary tab active item, links in views... they get the aria-current attribute, if the page is the current one. :+1: WFM.

**Comment by bugfolder** [comment]
Reviewed the code; LGTM.

**Comment by quicksketch** [comment]
Great little fix. Thanks @laryn, @indigoxela, and @bugfolder! I merged https://github.com/backdrop/backdrop/pull/4525 into 1.x and 1.26.x.
